src/components/form/InputSpan.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+class InputSpan extends Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    state = {
+        hidden: false
+    };
+
+    handleInputBlur = (e) => {
+        if (this.props.onloose(e)) {
```

Comment:
why you check this?

Reply:
Because we should change state only when validate is true